docs/FAQ.md

```diff
@@ -540,15 +540,6 @@ with something like `jj desc -m "Revert the merge of D into B`. Now, commit `@`
 undoes the merge of `D` into  `B`. If necessary, you can now rebase it
 elsewhere, e.g. `jj rebase -r @ -d main`.
 
-### How do I deal with divergent changes ('??' after the [change ID])?
```

Comment:
I think you should leave the FAQ entry, and link to the new doc for more details.

Reply:
I originally did that, but then realized we keep old versions of the documentation published.

Removing this question will only affect `/latest/` (which is labeled "pre-release" in the documentation version selector).

If someone needs a permanent link to documentation, I think they really should be linking to a specific version, not the "pre-release" documentation.

What do others think?

Reply:
I think people often link to the `/latest` version of the docs. for example, most links inside `jj`'s source code are to `/latest`.

Joy's point also seems reasonable to me; this would make the article more discoverable, and I don't see much harm in the FAQ question remaining.

So, I'd keep the question, even if it becomes a stub.

Reply:
I put back the question as a stub that links to the guide.